go/pvl/interp.go

```diff
@@ -418,25 +425,29 @@ func validateScript(g proofContextExt, script *scriptT, service keybase1.ProofTy
 
 // Run each script on each TXT record of each domain.
 // Succeed if any succeed.
-func runDNS(g proofContextExt, userdomain string, scripts []scriptT, mknewstate stateMaker) []libkb.ProofError {
+func runDNS(g proofContextExt, userdomain string, scripts []scriptT, mknewstate stateMaker, sigIDMedium string) libkb.ProofError {
 	domains := []string{userdomain, "_keybase." + userdomain}
 	var errs []libkb.ProofError
 	for _, d := range domains {
 		debug(g, "Trying DNS for domain: %v", d)
 
-		err := runDNSOne(g, d, scripts, mknewstate)
+		err := runDNSOne(g, d, scripts, mknewstate, sigIDMedium)
 		if err != nil {
 			errs = append(errs, err)
 		} else {
 			return nil
 		}
 	}
 
-	return errs
+	// Return only the error for the first domain error
+	if len(errs) == 0 {
+		return nil
+	}
+	return errs[0]
```

Comment:
Done, but I bet that most people only use the non`_keybase` version.

Now the simple error case where there is no TXT entry  looks like this.
```
✖ Proof for mikemaxim.com failed: Checked 1 TXT entries of mikemaxim.com, but didn't find signature keybase-site-verification=7SY8f7o5UQS0yXvfEOXUCkIluG3GK5iZyQIDjNvFrT8; DNS failure for _keybase.mikemaxim.com: lookup _keybase.mikemaxim.com on 192.168.1.1:53: no such host (code=201)
```